src/ui/display.c

```diff
@@ -150,35 +150,25 @@ bool ui_display_register_wallet_policy(
     for (size_t i = 0; i < wallet_header->n_keys; i++) {
         state->keys_info[i] = (*keys_info)[i];
 #ifdef SCREEN_SIZE_WALLET
-        const char labels[3][20] = {"internal", "external", "unspendable"};
+        const char labels[3][20] = {"Internal", "External", "Unspendable"};
 #else
-        const char labels[3][20] = {"ours", "theirs", "dummy"};
+        const char labels[3][20] = {"Our", "Their", "Dummy"};
 #endif
+        unsigned int num = 0;
         switch ((*keys_type)[i]) {
             case PUBKEY_TYPE_INTERNAL:
-                snprintf(state->keys_label[i],
-                         sizeof(state->keys_label[i]),
-                         "Key @%u, %s",
-                         i,
-                         labels[0]);
+                num = 0;
                 break;
             case PUBKEY_TYPE_EXTERNAL:
-                snprintf(state->keys_label[i],
-                         sizeof(state->keys_label[i]),
-                         "Key @%u, %s",
-                         i,
-                         labels[1]);
+                num = 1;
                 break;
             case PUBKEY_TYPE_UNSPENDABLE:
-                snprintf(state->keys_label[i],
-                         sizeof(state->keys_label[i]),
-                         "Key @%u, %s",
-                         i,
-                         labels[2]);
+                num = 2;
                 break;
             default:
                 LEDGER_ASSERT(false, "Unreachable code");
         }
+        snprintf(state->keys_label[i], sizeof(state->keys_label[i]), "%s key %u", labels[num], i);
```

Comment:
```suggestion
        snprintf(state->keys_label[i], sizeof(state->keys_label[i]), "%s key @%u", labels[num], i);
```

I would keep the `@`, because `@0`, `@1`, etc. refers to the corresponding numbers seen in the descriptor template just before. So it might be confusing to write `@2` in the descriptor template, and just `2` here. Hopefully it still fits?

Reply:
Finally that did not work well - the "Dummy" text was still too long:
<img width="128" height="64" alt="Dummy_with_@" src="https://github.com/user-attachments/assets/5e91ad41-339b-4ac2-8849-fcbbf6f7d92e" />

Finally I've come up with following version (no white-space between key and @):
`Dummy key@1 (1/3)`

<img width="128" height="64" alt="Dummy_with_@_without_whitespace" src="https://github.com/user-attachments/assets/0978f135-cedd-4ca8-a4ac-09f1decbd505" />

What do you think ?

Reply:
I don't love it, but better than truncating, so let's fix it this way for this release.

This will still truncate with more than 9 keys - which is probably rare in practice, but we support it.
So perhaps let's track in a new issue that there's still room for improvement (the ideal would probably be to remove the `(i/n)` part entirely, if the SDK can make it happen).

Reply:
The issue created: https://github.com/LedgerHQ/app-bitcoin-new/issues/394.